src/app/accounts/index.ts

```diff
@@ -10,11 +10,21 @@ export * from "./add-api-key-for-account"
 export * from "./get-api-keys-for-account"
 export * from "./disable-api-key-for-account"
 
+const accounts = AccountsRepository()
+
 export const getAccount = async (accountId: AccountId) => {
-  const accounts = AccountsRepository()
   return accounts.findById(accountId)
 }
 
+export const getWalletPublicIdFromUsername = async (
+  username: Username,
+): Promise<WalletPublicId | Error> => {
+  const account = await accounts.findByUsername(username)
```

Comment:
`username` will be renamed here in a followup PR right?

Reply:
yes

Reply:
assuming you mean renaming `username` to `handle`